parser.rb

```diff
@@ -1,13 +1,41 @@
 require 'pry'
 
 def kinda_like?(fuzzy_word, exact_word)
-  # implement with your code here
-  # breaking the method up into pieces is encouraged
+  if fuzzy_word == exact_word
+    return true
+  elsif fouls(fuzzy_word, exact_word) < 2
+    return true
+  else 
+    return false
+  end
 end
 
-#def another_private_method
-#end
+def fouls(w1,w2)
+  array_w2 = w2.chars
+  array_w1.each do |x|
+    array_w2.delete(x)
+  end
+  z = 0
+  wd1 = w1.chars
+  wd2 = w2.chars
+  z = 1 if wd1.last == wd2.last && wd1.first == wd2.first && w1.size != w2.size
+  array_w2.count + jumbled(w1,w2) + z
```

Comment:
Seems like `z` is derived from first and last letters, but, `jumbled` also is derived from first and last letters... is there a way to DRY this up?